eloquentjs/6_3_groups_iterable.js

```diff
@@ -0,0 +1,104 @@
+class GroupIterator {
+    [Symbol.iterator](set) {
```

Comment:
You shouldn't need `[Symbol.iterator]` here, this can just be a normal `constructor`. Also, you should avoid calling the argument `set` since it's a group.